dev-requirements.txt

```diff
@@ -1,58 +1,109 @@
 -i https://pypi.python.org/simple
+-e .
```

Comment:
I think this line is not needed.

Reply:
Isn't this a direct consequence of having `nucypher = {editable = true,path = "."}` listed as a dev dependency? The requirements files were generated automatically using the `pipenv lock --requirements` command.

Reply:
Indeed, this is a result of having the editable nucypher package listed in the `Pipfile` - however, we do not want *all* pip installations to be editable (and they do not need to be).  In short `Pipenv` is helpful, but not 100% accurate for our needs.  I suggest always keeping a close eye on machine-generated requirements files, as they almost are never exactly right.